torchaudio/models/_wavernn.py

```diff
@@ -192,3 +193,132 @@ def forward(self, specgram: Tensor) -> Tensor:
         upsampling_output = upsampling_output.squeeze(1)[:, :, self.indent:-self.indent]
 
         return upsampling_output, resnet_output
+
+
+class _WaveRNN(nn.Module):
+    r"""WaveRNN model based on the implementation from `fatchord <https://github.com/fatchord/WaveRNN>`_.
+
+    The original implementation was introduced in
+    `"Efficient Neural Audio Synthesis" <https://arxiv.org/pdf/1802.08435.pdf>`_.
+    The input channels of waveform and spectrogram have to be 1. The product of
```

Comment:
This is currently not quite true, since the model assumes there is no channel dimension. However, I prefer leaving this assumption as it is and changing the forward pass around it.

Reply:
I added the channel dimension in the waveform and spectrogram (also the input in `test_model.py` file). So the `unsqueeze(1)` and `squeeze(1)` operations are added in the forward function. A channel dimension assertion is also added before the `unsqueeze(1)` operation.